_data/grpcgettingstarteddnavswanlake.yml

```diff
@@ -0,0 +1,36 @@
+- title: Writing a gRPC Service with Ballerina
+  url: #
+  sublinks:
+    - title: Prerequisites
+      url: /learn/user-guide/getting-started/writing-a-grpc-service-with-ballerina/#prerequisites/
+    - title: Creating the Service Definition
+      url: /learn/user-guide/getting-started/writing-a-grpc-service-with-ballerina/#creating-the-service-definition/
+    - title: Implementation
+      url: /learn/user-guide/getting-started/writing-a-grpc-service-with-ballerina/#implementation/
+      sublinks:
+        - title: Developing a gRPC service
+          url: /learn/user-guide/getting-started/writing-a-grpc-service-with-ballerina/#developing-a-grpc-service/
+          active: developing-a-grpc-service
+          sublinks:
+            - title: Creating a Ballerina project
+              url: /learn/user-guide/getting-started/writing-a-grpc-service-with-ballerina/#developing-a-grpc-service/#creating-a-ballerina-project/
```

Comment:
@praneesha is it url correct. I added two hash contexts

Reply:
Should be: `/learn/writing-a-grpc-service-with-ballerina/#developing-a-grpc-service/#creating-a-ballerina-project/`